### DISCUSSION TITLE
Why did you choose non-box-drawing characters?

### DISCUSSION THREAD

**Discussion by beedell-roke** [jdhitsolutions/PSScriptTools#152] [Q&A] [answered]
`tree` uses them:

1.	```sh
	#!/usr/bin/env -S sh
	tree
	```
	
1.	```log
	PS /home/RokeJulianLockhart/Documents/webpage_frontend> tree
	.
	├── index.html
	└── server.js
	
	1 directory, 2 files
	```

...and they seem more readable than the examples in https://github.com/jdhitsolutions/PSScriptTools/blob/b345e01b3f8a268c9f3929af8f6681c754b13c23/docs/Show-Tree.md#examples.

**Comment by jdhitsolutions** [comment, marked as answer]
At the time I wrote the command, I went with the path of least resistance. I'm open to a PR if you want to improve this command.
